nifi-nar-bundles/nifi-mongodb-bundle/nifi-mongodb-processors/src/main/java/org/apache/nifi/processors/mongodb/GetMongo.java

```diff
@@ -126,6 +127,18 @@ public ValidationResult validate(final String subject, final String value, final
         .addValidator(StandardValidators.POSITIVE_INTEGER_VALIDATOR)
         .build();
 
+    static final AllowableValue YES_PP = new AllowableValue("yes", "Yes");
+    static final AllowableValue NO_PP  = new AllowableValue("no", "No");
+    static final PropertyDescriptor USE_PRETTY_PRINTING = new PropertyDescriptor.Builder()
+        .name("use-pretty-printing")
+        .displayName("Pretty Print Results JSON")
+        .description("Choose whether or not to pretty print the JSON from the results of the query. " +
+                "Choosing yes can greatly increase the space requirements on disk depending on the complexity of the JSON document")
+        .required(true)
+        .allowableValues(NO_PP, YES_PP)
+        .addValidator(Validator.VALID)
```

Comment:
This doesn't really hurt but just as an FYI, it's not really necessary to have a validator if allowable values are provided.